src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,37 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Status](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+Target audience for testing
+* Maintainer who has a workspace
+* Developing on nightly is optional
```

Comment:
Would it read better for you for this to say `*(optional)* Developing on nightly`?

Reply:
I like "*(optional)* Project depends on nightly toolchain" better